docs/topics/signalen-and-standard-for-public-code.md

```diff
@@ -1,68 +1,189 @@
+Contributions MUST pass all automated tests before they are admitted into the codebase. | yes |
+Contributions MUST be small. | yes | Each pull request contains a single feature or fix, but this policy is not explicit in CONTRIBUTING.md nor explictily checked by reviewer.
+The codebase MUST have active contributors. | yes | Amsterdam & VNG
+Source code test and documentation coverage SHOULD be monitored. | no | Coverage statics are not monitored but tests will fail when coverage is below a threshold.
+Policy and documentation MAY have testing for consistency with the source and vice versa. |  |
+Policy and documentation MAY have testing for style and broken links. |  |
 
 ## [Publish with an open license](https://standard.publiccode.net/criteria/open-licenses.html)
 
-* Good.
-    * License: [Mozilla Public License Version 2.0](../../LICENSE)
+- [ ]
+
+Requirement | meets | links and notes
+-----|-----|-----
+All code and documentation MUST be licensed such that it may be freely reusable, changeable and redistributable. | Yes | [Mozilla license](https://github.com/Amsterdam/signals/blob/master/LICENSE), would be nice to add SPDX license identifiers to files.
+Software source code MUST be licensed under an OSI-approved open source license. | yes | [MPL-2.0](https://opensource.org/licenses/MPL-2.0)
+All code MUST be published with a license file. | yes |
+All source code files in the codebase SHOULD include a copyright notice and a license header. | no |
+Codebases MAY have multiple licenses for different types of code and documentation. |  |
+Documentation MAY be published under Creative Commons licenses that are NOT ‘no derivatives’ or ‘non-commercial’. |  |
 
 ## [Use a coherent style](https://standard.publiccode.net/criteria/style.html)
 
-* Conforms to the PEP8 style guide, and is automatically checked.
+- [ ]
+
+Requirement | meets | links and notes
+-----|-----|-----
+Contributions MUST adhere to either a coding or writing style guide, either your own or an existing one that is advertised in or part of the codebase. | yes | CI fails when not adhering to code styles. There are some [design guidelines](https://github.com/Amsterdam/signals/blob/master/docs/topics/application-design.md#goals-and-design-principles).
+Contributions SHOULD pass automated tests on style. | yes |
+Your codebase SHOULD include inline comments and documentation for non-trivial sections. | mostly | There are many links to non-public resources/tickets.
+You MAY include sections in your style guide on understandable English. |  |
 
 ## [Pay attention to codebase maturity](https://standard.publiccode.net/criteria/advertise-maturity.html)
 
-* Migration to current REST API still in progress, completion status not currently published.
-* Publish plans/goals of next revision of the REST API.
+- [ ]
+
+Requirement | meets | links and notes
+-----|-----|-----
+A codebase MUST be versioned. | yes | Semantic versioning and tagged as such.
```

Comment:
work-in-progress as noted above

Reply:
sounds like we might need to expand the notes on this one.

Reply:
Done